-------------------------------------------------------------------------------
Test set: org.apache.hadoop.ozone.om.TestScmSafeMode
-------------------------------------------------------------------------------
Tests run: 5, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 144.645 s <<< FAILURE! - in org.apache.hadoop.ozone.om.TestScmSafeMode
testSafeModeOperations(org.apache.hadoop.ozone.om.TestScmSafeMode)  Time elapsed: 13.247 s  <<< ERROR!
VOLUME_NOT_FOUND org.apache.hadoop.ozone.om.exceptions.OMException: Volume doesn't exist
	at org.apache.hadoop.ozone.om.BucketManagerImpl.createBucket(BucketManagerImpl.java:130)
	at org.apache.hadoop.ozone.om.OzoneManager.createBucket(OzoneManager.java:2089)
	at org.apache.hadoop.ozone.om.TestScmSafeMode.testSafeModeOperations(TestScmSafeMode.java:159)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.FailOnTimeout$StatementThread.run(FailOnTimeout.java:74)

